site/releases.md

```diff
@@ -27,6 +27,14 @@ Client Guide | API docs
 
 ## News
 
+### XX November, 2017: Release 4.5.1 available
+
+This is the sixth release of BookKeeper as an Apache Top Level Project!
+
+The 4.5.1 release is a bugfix release.
```

Comment:
A quick summary of the highlights would be good here.

Reply:
isn't "The technical details of this release are summarized below" enough ?